tests/test/test_rks_administration_api.py

```diff
@@ -193,6 +193,14 @@ def test_multiple_init(self, init_api):
             excinfo.value.status == 409
         ), "Error with multiple init, must return 409 code"
 
+        # test a call to init endpoint without content-type header fix #1
+        response = requests.post(
+            rks_url + "/rks/v1/init",
+            headers={"X-Vault-Token": utils.ADMIN_TOKEN},
+            verify=False,
+        )
+        assert response.status_code == 409
```

Comment:
Can we put this test in a new test function ?
Like test_post_without_content_type